src/olympia/activity/models.py

```diff
@@ -655,11 +656,11 @@ def __html__(self):
     def author_name(self):
         """Name of the user that triggered the activity.
 
-        If it's a reviewer action that will be shown to developers, the
-        `reviewer_name` property is used if present, otherwise `name` is
-        used."""
+        If it's a reviewer action that will be shown to developers, use a generic
+        Review team name"""
         if self.action in constants.activity.LOG_REVIEW_QUEUE_DEVELOPER:
-            return self.user.reviewer_name or self.user.name
+            if is_user_any_kind_of_reviewer(self.user):
```

Comment:
Per comment below, you should remove this check, if will just create a bunch of additional queries (~ one per item).

Instead, I would suggest to just return the generic name always in this property for reviewer actions, and use something completely different when you know ahead of time you're showing content for a reviewer.

Reply:
I am not sure I follow this. As far as I understand, this method is called both for reviewer and developer replies. If the reply comes from a reviewer, we want to use _Firefox Add-on Review Team_ instead, but if the sender is a developer, it should use the developer name.

Reply:
You can know that with the `action` and display the reviewer name for reviewer actions and the user name for developer actions. If that's all you think we need, you can use that (I think `review_action` property on the action should be enough) and stop there.

What I thinking about is, in reviewer or admin contexts, you might want to see which reviewer it actually was. If that's the case you would need to use something different when displaying the activity, in those contexts, to avoid the generic name and show the true user name.

Reply:
Ah, but is `author_name` even used in reviewer or admin contexts?

Reply:
I don't know, maybe not (which would help), you'll have to find out :)